wfl/DBManager.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using System.Configuration;
+using System.Threading.Tasks;
+using MongoDB.Driver;
+using MongoDB.Bson;
+using wfl.Models;
+
+namespace wfl
+{
+    public class DBManager
+    {
+        private static string connectionString;
+        public static MongoClient client;
+        public static IMongoDatabase db;
+
+        static DBManager()
+        {
+            connectionString = ConfigurationManager.ConnectionStrings["databaseConnectionString"].ConnectionString;
+            client = new MongoClient(connectionString);
+            db = client.GetDatabase("WFL");
+        }
+
+        public static async Task InsertRestaurantAsync(Restaurant r)
+        {
+            IMongoCollection<Restaurant> collection = db.GetCollection<Restaurant>("restaurants");
+            await collection.InsertOneAsync(r);
+        }
+        public static async Task<Restaurant> UpdateRestaurantNameAsync(ObjectId id, string name)
```

Comment:
This is a mistake on my part. DBManager should no longer exist. I am going to get rid of it now.